stdlib/public/runtime/Heap.cpp

```diff
@@ -73,7 +73,11 @@ static inline malloc_zone_t *DEFAULT_ZONE() {
 // the use of AlignedAlloc. This allows manually allocated to memory to always
 // be deallocated with AlignedFree without knowledge of its original allocation
 // alignment.
-//
+static size_t computeAlignment(size_t alignMask) {
+  return (alignMask == ~(size_t(0))) ? _swift_MinAllocationAlignment
+                                     : alignMask + 1;
+}
+
```

Comment:
Extract this helper to reduce code duplication.  Also: relevant comment is now right next to the code.